robots/preservation/create_moab.rb

```diff
@@ -0,0 +1,26 @@
+
+module Preservation
+  # Update or create moab object
+  class CreateMoab < Base
+    ROBOT_NAME = 'creaet-moab'.freeze
+
+    def initialize(opts={})
+      super(REPOSITORY, WORKFLOW_NAME, ROBOT_NAME, opts)
+    end
+
+    def perform(druid)
+      storage_object = Moab::StorageServices.find_storage_object(druid, true)
+      storage_object.object_pathname.mkpath
+      create_moab(storage_object)
+    end
+
+    def create_moab(storage_object)
+      new_version = storage_object.ingest_bag
+      result = new_version.verify_version_storage
+      unless result.verified
+        LyberCore::Log.info result.to_json(false)
+        raise(ItemError, "Failed verification")
```

Comment:
can we have a little more info?  Is it easy to get which version failed verification?  is it easy to get info about why?  or was it already logged?